packages/web-app-files/tests/unit/store/actions.spec.ts

```diff
@@ -0,0 +1,108 @@
+import actions from '../../../src/store/actions'
+import { spaceRoleManager, ShareTypes, Share } from 'web-client/src/helpers/share'
+import { mockDeep } from 'jest-mock-extended'
+import { OwnCloudSdk } from 'web-client/src/types'
+
+jest.mock('../../../src/helpers/resources')
+
+const stateMock = {
+  commit: jest.fn(),
+  dispatch: jest.fn(),
+  getters: {},
+  rootGetters: {}
+}
+
+describe('vuex store actions', () => {
+  beforeEach(() => {
+    // because of the $gettext warnings
+    jest.spyOn(console, 'warn').mockImplementation(() => undefined)
```

Comment:
What gettext warnings? Can we fix them instead?

Reply:
`No translations found for en`.

The actions use `$getttext` from here: https://github.com/owncloud/web/blob/master/packages/web-app-files/src/gettext.js. Honestly, I'm not sure if this is the right way to go in general. If I remember correctly, we want to get rid of this file anyway in the future. Hence I figured to not spend more time finding another solution 😅

Reply:
Yeah, I'm just not a great fan of hiding console output ... I've spent way too much time in my life looking for missing output, when I wanted to debug something and probably even worse I'm afraid we might miss something more interesting. :-/

I've approved the PR, so if you decide it's not worth it, then just merge it

Reply:
I changed the way how it is mocked to `jest.mock('../../../src/gettext')`. This way it only affects `$gettext` and not all console warnings :)